src/scene/shader-lib/chunks-wgsl/common/frag/encode.js

```diff
@@ -0,0 +1,46 @@
+export default /* glsl */`
```

Comment:
Is there no wgsl syntax highlighter yet?

Reply:
there is and using it on few chunks, let me update the rest, cheers